linux_os/guide/system/accounts/accounts-pam/display_login_attempts_disa/rule.yml

```diff
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+title: 'Ensure PAM Displays Last Logon/Access Notification'
+
+description: |-
+    To configure the system to notify users of last logon/access
+    using <tt>pam_lastlog</tt>, add or correct the <tt>pam_lastlog</tt>
+    settings in
+    <tt>/etc/pam.d/postlogin</tt> to read as follows:
+    <pre>session     required      pam_lastlog.so showfailed</pre>
+
+rationale: |-
+    Users need to be aware of activity that occurs regarding
+    their account. Providing users with information regarding the number
+    of unsuccessful attempts that were made to login to their account
+    allows the user to determine if any unauthorized activity has occurred
+    and gives them an opportunity to notify administrators.
+
+severity: low
+
+identifiers:
+    cce@rhel7: CCE-27275-7
+    cce@rhel8: CCE-80788-3
+    cce@rhel9: CCE-83560-3
```

Comment:
Instead moving CCE identifiers around, use identifiers available from this pool: https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt